### PR TITLE
Experience Brand kit updates

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

update legal licenses summary

add info on brand voice 

### Tested

Screen Shot 2020-03-11 at 11 17 57 AM
Screen Shot 2020-03-11 at 11 18 05 AM
Screen Shot 2020-03-11 at 11 18 28 AM

### Other changes

remove the `a` on the selector as it makes it too specific

remove duplicate normalization css

### Related issues

- Fixes https://github.com/celo-org/celo-labs/issues/345
FIXES https://github.com/celo-org/celo-labs/issues/344

### Backwards compatibility

_Brief explanation of why these changes are/are not backwards compatible._

### CODE REVIEW

File: packages/web/src/brandkit/Intro.tsx [hunk 1]
```diff
@@ -29,11 +33,44 @@ const Overview = withNamespaces(NameSpaces.brand)(function _Overview({ t }: I18n
...
   )
 })
+
+function BrandVoice() {
+  const { t } = useTranslation(NameSpaces.brand)
+
+  return (
+    <View style={brandStyles.gap}>
+      <SectionTitle>{t('home.brandVoice')}</SectionTitle>
+      {/* <Text style={[fonts.h5, standardStyles.elementalMarginBottom]}>{t('home.brandVoice')}</Text> */}
```
Comment (jmrossy) [inline packages/web/src/brandkit/Intro.tsx]: Nit: remove?

File: packages/web/src/brandkit/Intro.tsx [hunk 2]
```diff
@@ -29,11 +33,44 @@ const Overview = withNamespaces(NameSpaces.brand)(function _Overview({ t }: I18n
...
+      <Text style={fonts.p}>
```
Comment (jmrossy) [inline packages/web/src/brandkit/Intro.tsx:62]: Wonder if we could just make a `<P>` tag. Not sure how often you use this, up to you

**Review by jmrossy — APPROVED** [review]
LGTM, some optional comments below

File: packages/web/src/brandkit/Intro.tsx (hunk 1 above)
Comment (aaronmgdr) [reply packages/web/src/brandkit/Intro.tsx]: aye.

File: packages/web/src/brandkit/Intro.tsx (hunk 2 above)
Comment (aaronmgdr) [reply packages/web/src/brandkit/Intro.tsx:62]: on the fence

**Pull request merged**